sonar-db/src/main/java/org/sonar/db/permission/PermissionRepository.java

```diff
@@ -155,10 +155,15 @@ public void applyPermissionTemplate(DbSession session, String templateUuid, long
     }
   }
 
-  public void grantDefaultRoles(DbSession session, long componentId, String qualifier) {
-    ResourceDto resource = dbClient.resourceDao().selectResource(componentId, session);
-    String applicablePermissionTemplateKey = getApplicablePermissionTemplateKey(session, resource.getKey(), qualifier);
-    applyPermissionTemplate(session, applicablePermissionTemplateKey, componentId);
+  // TODO Add UT
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1135)